### PR TITLE
converted from g_test_add() to g_test_add_func()

### PR DESCRIPTION
**Pull request by sashreek1**
corrected the previous syntax and also modified the required functions to enable compatibility with g_test_add_func()

### CODE REVIEW

**Comment by sashreek1** [comment]
I dida run ```ninja test ``` before submitting so the code should have been formatted right? Moreover all the tests had passed.

File: modulemd/tests/test-dirty.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 # For more information on the license, see COPYING.
 # For more information on free software, see
 # <https://www.gnu.org/philosophy/free-sw.en.html>.
+#comment
```
Comment (sgallagher) [inline modulemd/tests/test-dirty.py]: What is the point of this?

Comment (sashreek1) [reply modulemd/tests/test-dirty.py]: I just removed it i will  be pushing it

**Comment by sashreek1** [comment]
I have been running ```ninja test``` before every push, yet it returns a failure

**Pull request merged**